src/lib.rs

```diff
@@ -299,4 +357,49 @@ mod tests {
         assert!(env::var_os("LISTEN_PID").is_none());
         assert!(env::var_os("LISTEN_FDS").is_none());
     }
+
+    #[test]
+    fn watchdog_enabled() {
+        // test original logic: https://github.com/systemd/systemd/blob/f3376ee8fa28aab3f7edfad1ddfbcceca5bc841c/src/libsystemd/sd-daemon/sd-daemon.c#L632
```

Comment:
I didn't open this file, so I don't know how close your implementation is to the original wrt. unset `WATCHDOG_PID`, but I'm slightly wary of potential licensing implications here. This crate is licensed as MIT/Apache-2.0, while `libsystemd` is LGPL.

Your implementation of `watchdog_enabled` seems almost trivial, but I'm wary to look, or even link to the `systemd` implementation of it. And if we do add a link to it, it should probably be in the implementation, not the test code.

Reply:
agree, we should better remove the link.